scalafmt-tests/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -76,14 +76,12 @@ class CliTest extends FunSuite with DiffAssertions {
   def gimmeConfig(string: String): ScalafmtConfig =
     Config.fromHoconString(string).get
 
-  test("scalafmt -i --file tmpFile") {
+  test("scalafmt tmpFile") {
```

Comment:
Can we add or modify a test  to check that we can pass in multiple files like `scalafmt Foo.scala Bar.scala`?